distribution/docs/src/main/resources/content/_appReferences/mg-admin.adoc

```diff
@@ -6,12 +6,9 @@
 
 == {title} Application Reference
 
-The ${ddf-admin} Application contains components that are integral for the installation and configuration of ${branding} applications.
+The ${ddf-admin} Application contains components that are integral for the configuration of ${branding} applications.
 
-It contains various services and interfaces that allow administrators more control over their systems and enhances administrative capabilities when installing and managing ${branding}.
-
-The ${ddf-admin} application contains an application service that handles all operations that are performed on applications.
-This includes adding, removing, starting, stopping, and showing status.
+It contains various services and interfaces that allow administrators more control over their systems and enhances administrative capabilities.
```

Comment:
✏️ I don't think "more control" makes sense any more now, as there's only on admin UI. Can you cut it?

Reply:
just realized this might be unclear. I just mean eliminate the "more"